### PR TITLE
Add support pack to debug info

### PR DESCRIPTION
**Pull request by mbtools**

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_page_debuginfo.clas.abap [hunk 1]
```diff
@@ -161,6 +162,8 @@ CLASS zcl_abapgit_gui_page_debuginfo IMPLEMENTATION.
       iv_typ = zif_abapgit_html=>c_action_type-url ).
     ri_html->add( '</div>' ).
 
+    SELECT SINGLE extrelease FROM cvers INTO lv_sp WHERE component = 'SAP_BASIS' ##SUBRC_OK.
```
Comment (larshp) [inline src/ui/zcl_abapgit_gui_page_debuginfo.clas.abap:165]: the `SELECT` seems a bit malplaced here, perhaps move it to a method in `zcl_abapgit_factory=>get_environment( )` ?

**Comment by larshp** [comment]
return the fields in a structure? will make the method call shorter

**Comment by mbtools** [comment]
it's early ;-). yeah, will do

**Review by larshp — APPROVED** [review]
LGTM

**Pull request merged**